src/test/scala/org/constellation/primitives/TipServiceTest.scala

```diff
@@ -30,7 +31,7 @@ class TipServiceTest extends FunSpecLike with IdiomaticMockito with ArgumentMatc
 
     it("limits maximum number of tips") {
       val limit = 6
-      val concurrentTipService = new TrieBasedTipService(limit, 10, 2, 30)
+      val concurrentTipService = new TrieBasedTipService(limit, 10, 2, 30, null)
```

Comment:
What about `Option`/`None` instead of `null`?

Reply:
Ok, it's a test, nvm

Reply:
Its a test and actor ref for that particular test is not needed. Let's leave it as there is no danger.